certified-connectors/JG Integrations/apiDefinition.swagger.json

```diff
@@ -0,0 +1,517 @@
+                    "b8",
+                    "b9",
+                    "b10",
+                    "c0",
+                    "c1",
+                    "c2",
+                    "c3",
+                    "c4",
+                    "c5",
+                    "c6",
+                    "c7",
+                    "c8",
+                    "c9",
+                    "c10",
+                    "ra0",
+                    "ra1",
+                    "ra2",
+                    "ra3",
+                    "ra4",
+                    "sra0",
+                    "sra1",
+                    "sra2",
+                    "sra3",
+                    "sra4",
+                    "letter",
+                    "half-letter",
+                    "legal",
+                    "ledger",
+                    "tabloid",
+                    "executive",
+                    "folio",
+                    "commercial #10 envelope",
+                    "catalog #10 1/2 envelope",
+                    "8.5x11",
+                    "8.5x14",
+                    "11x17",
+                    "]"
+                  ]
+                }
+              },
+              "required": [
+                "html",
+                "landscape",
+                "name",
+                "pagesize"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/text/file_string": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "JSON Response",
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "type": "object",
+              "properties": {
+                "output": {
+                  "type": "string",
+                  "description": "Cleaned string",
+                  "title": "Cleaned string"
+                }
+              }
+            }
+          }
+        },
+        "summary": "File Friendly String",
+        "description": "Remove non-alphanumeric and - ( ) _ characters",
+        "operationId": "FILE_STRING",
+        "parameters": [
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "pattern": {
+                  "type": "string",
+                  "description": "pattern",
+                  "title": "",

```

Comment:
@jglassonnz As I have already made a comment for empty properties: Empty property values can be removed until unless you have a specific reason to keep them.